specification/resources/resource-manager/Microsoft.Resources/stable/2019-07-01/resources.json

```diff
@@ -601,6 +601,68 @@
         }
       }
     },
+    "/providers/Microsoft.Resources/deployments/{deploymentName}/whatIf": {
+      "post": {
+        "tags": [
+          "Deployments"
+        ],
+        "operationId": "Deployments_WhatIfAtTenantScope",
+        "description": "Returns changes that will be made by the deployment if executed at the scope of the tenant group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeploymentNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DeploymentWhatIf"
```

Comment:
One slight difference is for Tenant/MG/Sub level deployment the "location" property is required.

Reply:
Good catch. Will add a `ScopedDeploymentWhatIf` for the API version.

Reply:
Done.